backend/typescript/services/implementations/reviewService.ts

```diff
@@ -207,6 +208,16 @@ class ReviewService implements IReviewService {
     };
   }
 
+  static getUserDetails(review: PgReview): User {
```

Comment:
haha same as my PR, last term we didn't write any documentation for the functions. We should probably start this term though to explain what the function does. 

It's a nit pick too but you might wanna add some documentation here to set the standard for future (perhaps more complex) function?

Reply:
good point - didn't realize i missed this!